js/game.js

```diff
@@ -0,0 +1,83 @@
+class Game {
+  constructor(ctx, width, height, player) {
+    this.ctx = ctx;
+    this.width = width;
+    this.height = height;
+    this.player = player;
+    this.intervalId = null;
+    this.frames = 0;
+    this.enemies = [];
+    this.points = 0;
+  }
+  start() {
+    this.intervalId = setInterval(this.update, 1000 / 60);
+  }
+  update = () => {
+    this.frames += 1;
+    this.clear();
+    this.player.newPos();
+    this.player.draw();
+    this.changeEnemies();
+    this.checkGameOver();
+    this.countPoints();
+    this.drawScore();
+  };
+  stop() {
+    clearInterval(this.intervalId);
+  }
+  clear() {
+    this.ctx.clearRect(0, 0, this.width, this.height);
+  }
+  changeEnemies() {
+    this.enemies.forEach((el) => {
+      el.y += 10;
+      el.draw();
+    });
+
+    if (this.frames % 30 === 0) {
+      let randomSize = Math.floor(Math.random() * 250 - 100) + 100;
+      let randomX =
```

Comment:
You should predefine the height of your enemy, randomize the width of them, and the x where they will start appearing. Always taking in consideration that you need to leave space for the car